protocol/x/listing/keeper/listing.go

```diff
@@ -0,0 +1,35 @@
+package keeper
+
+import (
+	"encoding/json"
+
+	"golang.org/x/xerrors"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/dydxprotocol/v4-chain/protocol/x/listing/types"
+)
+
+// Function to set permissionless listing flag in module store
+func (k Keeper) SetPermissionlessListingEnable(ctx sdk.Context, enable bool) error {
+	store := ctx.KVStore(k.storeKey)
+	b, err := json.Marshal(enable)
```

Comment:
any particular reason why we are marshaling using json instead using the encoder (ie `k.cdc.MustMarshal`)? is it because it's not a proto but just a bool?

i didn't find any particular use cases of [json marshaling in our codebase](https://github.com/search?type=code&q=repo%3Adydxprotocol%2Fv4-chain+json.Marshal) for things that are stored in store

Reply:
That's exactly right. We can't use protobuf encoding since its not a proto which feels like overkill for a single boolean. 
We don't have another use case for it but do you see any problems with json encoding/decoding?